Gem/Code/Source/CullingAndLodExampleComponent.cpp

```diff
@@ -363,7 +363,7 @@ namespace AtomSampleViewer
                 ImGui::RadioButton("4", &m_cascadeCount, 4);
             if (cascadesChanged)
             {
-                m_directionalLightFeatureProcessor->SetCascadeCount(m_directionalLightHandle, m_cascadeCount);
+                m_directionalLightFeatureProcessor->SetCascadeCount(m_directionalLightHandle, static_cast<uint16_t>(m_cascadeCount));
```

Comment:
`m_cascadeCount` can get the same treatment as `m_predictionSampleCount` and `m_filteringSampleCount`